testing/src/cypress/page-objects/Institution-objects/InstitutionProfileObject.ts

```diff
@@ -8,27 +10,43 @@ export default class InstitutionProfileObject {
   }
 
   legalOperatingNameInputText() {
-    return cy.get("[data-cy='legalOperatingName']");
+    return this.getElementByCyId("legalOperatingName");
+  }
+
+  operatingName() {
+    return this.getElementByCyId("operatingName");
+  }
```

Comment:
Is it worth to create a method just to get an element?

Reply:
Yeah - the point is to be more structured and consistent in keeping all the individual page elements in a single file. This is also to avoid putting in the element identifiers directly in the test case, so if this element is being validated in more than one case, we can reduce code redundancy